Packs/Expanse/TestPlaybooks/playbook-Expanse-Test.yml

```diff
@@ -257,11 +339,17 @@ inputs:
     simple: 74.142.119.130
   required: false
   description: 'IP address '
+  playbookInputQuery: null
 - key: domain
   value:
     simple: atlas.enron.com
   required: false
   description: domain
+  playbookInputQuery: null
+- key: common_name
+  value:
+    simple: '*.enron.com'
+  required: false
+  description: A partial certificate common name
+  playbookInputQuery: null
 outputs: []
-description: ''
```

Comment:
revert these changes please, `description` and `fromversion` should still be here